tests/tests_pytorch/callbacks/progress/test_rich_progress_bar.py

```diff
@@ -416,3 +417,17 @@ def test_rich_progress_bar_padding():
     train_description = progress_bar._get_train_description(current_epoch=0)
     assert "Epoch 0/0" in train_description
     assert len(progress_bar.validation_description) == len(train_description)
+
+
+def test_rich_progress_bar_can_be_pickled():
+    bar = RichProgressBar()
+    trainer = Trainer(fast_dev_run=True, callbacks=[bar], max_steps=1)
+    model = BoringModel()
+
+    pickle.dumps(bar)
+    trainer.fit(model)
+    pickle.dumps(bar)
+    trainer.test(model)
```

Comment:
How could you forget about validate????!!!

Reply:
I copied the test function from the tqdm tests. It was probably written before validate even existed.